netbox/netbox.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2018-present the CoreDHCP Authors. All rights reserved
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package netbox
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+type addr struct {
```

Comment:
Can this be replaced with net.IPNet?

https://golang.org/pkg/net/#IPNet

Reply:
I couldn't remember the name and spent 20 minutes trying to find it.... thanks

Reply:
done